apis/Google.Cloud.Firestore/docs/history.md

```diff
@@ -0,0 +1,15 @@
+# Version history
+
+# Version 1.1.0, released 2019-12-03
```

Comment:
These are missing de links. Maybe add the links or delete the references to the issues/PRs

Reply:
Will add the links. These ones were copied from a commit.